scripts/legacy_expression_tool/README.md

```diff
@@ -225,8 +234,10 @@ All                                                    1        1   2
 #### Execution with `expressionlanguage` set to `jexlall`
 
 When executing the script with `expressionlanguage` set to `jexlall`, the script will look for all the documents
-containing legacy expressions or the parameter `expressionLanguage`. This would change the number of documents found,
-and the statistics will be different. Running the script with the option set to `jexl` would not have impact on  the statistics. 
+containing legacy expressions (in some of the expression capable fields) or the existence of the `expressionLanguage` field.
```

Comment:
It would be great if the list of "expression capable fields" could be added here.

Reply:
The script looks for expressions in the following fields:

- active.expression
- active.entity_name
- active.reverse
- attributes.expression
- attributes.entity_name
- attributes.expression
- commands.expression
- endpoint
- entityNameExp
- explicitAttrs